dagger-functions/src/main/java/io/odpf/dagger/functions/udfs/scalar/ArrayOperate.java

```diff
@@ -38,16 +42,17 @@ public void open(FunctionContext context) throws Exception {
     /**
      * Given an Object Array, this UDF performs basic functions on the Array.
      *
-     * @param arrayElements the array elements
      * @param operationType the operation type
      * @param inputDataType the input data type
+     * @param arrayElements the array elements
      * @return the result of the aggregate
      */
-    public Object[] eval(Object[] arrayElements, String operationType, String inputDataType) {
+    public @DataTypeHint(value = "RAW", bridgedTo = List.class)
```

Comment:
Could not find any Dagger using this and using something like `unnest` or `cardinality`. So changing the contract to Object[] to List<Object> should be fine here. However, I am trying a few things to support Object[] in the UDF contract. If that works we should change the contract to the old ones.